snapcraft/plugins/python.py

```diff
@@ -217,18 +217,22 @@ def build(self):
             self._python_major_version, stage_dir=self.project.stage_dir,
             install_dir=self.installdir)
 
+    def _find_file(self, *, filename: str) -> str:
+        # source-subdir defaults to ''
+        for basepath in [self.builddir, self.sourcedir]:
```

Comment:
Builddir includes the subdir, FYI.

Reply:
yeah, that's in the last push :-/

Reply:
Would using `self.build_basedir` not be cleaner than special-casing the sourcedir?